Description: Export the right aliases for auto-load on open.
Author: Scott James Remnant <scott@ubuntu.com>
--- a/drivers/old_st7554.c
+++ b/drivers/old_st7554.c
@@ -1386,4 +1386,5 @@
 MODULE_AUTHOR("Smart Link Ltd.");
 MODULE_DESCRIPTION("ST7554 USB Smart Link Soft Modem driver.");
 MODULE_LICENSE("Smart Link Ltd.");
+MODULE_ALIAS_CHARDEV_MAJOR(MY_MAJOR);
 
--- a/drivers/st7554.c
+++ b/drivers/st7554.c
@@ -1434,3 +1434,4 @@
 MODULE_DESCRIPTION("ST7554 USB Smart Link Soft Modem driver.");
 //MODULE_LICENSE("Smart Link Ltd.");
 MODULE_LICENSE("Dual BSD/GPL");
+MODULE_ALIAS_CHARDEV_MAJOR(243);
--- a/drivers/amrmo_init.c
+++ b/drivers/amrmo_init.c
@@ -741,6 +741,7 @@
 MODULE_AUTHOR("Smart Link Ltd.");
 MODULE_DESCRIPTION("SmartLink HAMR5600,SmartPCI56/561 based modem driver");
 MODULE_LICENSE("Smart Link Ltd.");
+MODULE_ALIAS_CHARDEV_MAJOR(AMRMO_MAJOR);
 
 
 static int __init amrmo_init(void)
